packages/core/src/core/communication.ts

```diff
@@ -37,38 +41,17 @@ export async function refreshSession(
   return axios.get<string>(url).then((res) => res.data);
 }
 
-export type RepoData = Record<
-  string,
-  { role: roles.Roles | Record<string, roles.Roles>; dbid: string }
->;
-const RepoDataValidator = t.record(
-  t.string,
-  t.type({
-    role: roles.RolesValidator,
-    dbid: t.string,
-  })
-);
-export type UserInfo = {
-  userId: string;
-  email: string;
-  type: string;
-  repositories: RepoData;
-};
-
-export function maybeParseRepoData(repos?: string | RepoData): RepoData {
+export function maybeParseRepoData(
```

Comment:
I can't remember where this function was used, but if it's parameters and return type have changed so should the name,